materials/databases-intro-sql.md

```diff
@@ -37,20 +37,44 @@ language: SQL
 	  * Access - commonly used, GUI
 	  * PostgreSQL - fast/powerful, lots of users
 
-> Open and preview SQLite with `portal_mammals.sqlite`.
-
-### Key features of database management systems
-
-* Data is separate from manipulations of the data
-* Tables - store the data
-* Queries - store questions about the data
-    * If we update the data, the query asks the same question of the new data.
-
 ### Relational databases
 
 * Data is stored in tables.
     * One table per type of data
     * Tables can be linked together to combine information.
+
+> Open SQLite Manager
+> Walk through importing surveys table
+
+1. Download https://ndownloader.figshare.com/files/2292172
```

Comment:
Links in the materials are normally used to open something on a separate machine so they need to be written out so they can be typed in.